.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--
```

Comment:
Not sure this belongs to this PR, as we might want to discuss the template on another PR. But if no one objects it, I'm fine keeping it here.

Reply:
It's just the continuity, we have code format, lint. Now unit test so I add PR template

Reply:
Yes, but you must understand, that, ahem, this is an unrelated change to adding the unit test framework :)

Reply:
I will fix this

Reply:
It's fix take a look to pr subject

Reply:
Cool :)